packages/gatsby-plugin-gatsby-cloud/src/__tests__/routes.js

```diff
@@ -0,0 +1,102 @@
+import * as fs from "fs-extra"
+import * as path from "path"
+import * as os from "os"
+const { onPostBuild } = require(`../gatsby-node`)
+
+describe(`Routes IPC`, () => {
+  let tmpDir
+  beforeAll(async () => {
+    tmpDir = await fs.mkdtemp(path.join(os.tmpdir(), `item-dir`))
```

Comment:
Normally I would say that you should mock fs but most of this plugin is not mocking fs so if you can change the tmpdir name this would be good to go.

```suggestion
    tmpDir = await fs.mkdtemp(path.join(os.tmpdir(), `gatsby-plugin-gatsby-cloud-item-dir`))
```